ni_measurementlink_service/_internal/parameter/serializer.py

```diff
@@ -67,6 +75,16 @@ def serialize_parameters(
             parameter_metadata.type,
             parameter_metadata.repeated,
         )
+        # Convert enum parameters to their underlying value.
+        if "ni/type_specialization" in parameter_metadata.annotations:
+            if (
+                parameter_metadata.annotations["ni/type_specialization"]
+                == TypeSpecialization.Enum.value
+            ):
```

Comment:
```suggestion
        if parameter_metadata.annotations.get("ni/type_specialization") == TypeSpecialization.Enum.value:
```
and fix indent

Reply:
Note that the indentation becomes a bit wonky again after running `poetry run ni-python-styleguide fix`. 

![image](https://github.com/ni/measurementlink-python/assets/103216564/58345634-6736-430c-9e5f-e050ac8cb516)